src/db_queries/fields.ts

```diff
@@ -14,6 +14,7 @@ export class Fields {
     autoBind(this);
   }
 
+  // not considered for transactions yet
```

Comment:
I think this comment is unnecessary.

Reply:
It was not meant to be pushed. Removed them.